tests/Examples/V2/ExampleTestsUtils.cs

```diff
@@ -0,0 +1,310 @@
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Ads.GoogleAds.Examples;
+using Google.Ads.GoogleAds.Lib;
+using Google.Ads.GoogleAds.Tests.V2;
+using Google.Ads.GoogleAds.V2.Common;
+using Google.Ads.GoogleAds.V2.Enums;
+using Google.Ads.GoogleAds.V2.Resources;
+using Google.Ads.GoogleAds.V2.Services;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using static Google.Ads.GoogleAds.V2.Enums.AdGroupAdStatusEnum.Types;
+using static Google.Ads.GoogleAds.V2.Enums.AdGroupCriterionStatusEnum.Types;
+using static Google.Ads.GoogleAds.V2.Enums.AdvertisingChannelTypeEnum.Types;
+using static Google.Ads.GoogleAds.V2.Enums.BudgetDeliveryMethodEnum.Types;
+using static Google.Ads.GoogleAds.V2.Enums.CampaignStatusEnum.Types;
+using static Google.Ads.GoogleAds.V2.Enums.KeywordMatchTypeEnum.Types;
+using static Google.Ads.GoogleAds.V2.Resources.Campaign.Types;
+
+namespace Google.Ads.GoogleAds.Tests.Examples.V2
+{
+    internal class ExampleTestsUtils : TestUtils
+    {
+        /// <summary>
+        /// Creates the budget for the campaign.
+        /// </summary>
+        /// <param name="client">The Google Ads client.</param>
+        /// <returns>The resource name of the newly created campaign budget.</returns>
+        internal static string CreateBudget(GoogleAdsClient client)
+        {
+            // Create the campaign budget.
+            CampaignBudget budget = new CampaignBudget()
+            {
+                Name = "Interplanetary Cruise Budget #" + ExampleUtilities.GetRandomString(),
+                DeliveryMethod = BudgetDeliveryMethod.Standard,
+                AmountMicros = 500000
+            };
+
+            // Create the operation.
+            MutateOperation mutateOperation = new MutateOperation()
+            {
+                CampaignBudgetOperation = new CampaignBudgetOperation()
+                {
+                    Create = budget,
+                },
+            };
+
+            return ExecuteOperations(client, new[] { mutateOperation }).First()
+                .CampaignBudgetResult.ResourceName;
+        }
+
+        /// <summary>
+        /// Creates the campaign.
+        /// </summary>
+        /// <param name="client">The Google Ads client.</param>
+        /// <param name="budgetResourceName">The resource name for the budget.</param>
+        /// <returns>The resource name of the newly created campaign.</returns>
+        internal static string CreateCampaign(GoogleAdsClient client, string budgetResourceName)
+        {
+            Campaign campaign = new Campaign()
+            {
+                Name = "Interplanetary Cruise #" + ExampleUtilities.GetRandomString(),
+                AdvertisingChannelType = AdvertisingChannelType.Search,
+
+                // Recommendation: Set the campaign to PAUSED when creating it to prevent
```

Comment:
nit: I would remove unnecessary comments and lines of code as these methods are for util purpose not code examples.

Reply:
Done